seed-job/buildtemplate.yaml

```diff
@@ -112,3 +112,11 @@ parameters:
   displayName: OpenShift namespace
   name: NAMESPACE
   required: true
+- description: From address to be used when sending email
+  displayName: From email address
+  name: FROM_ADDRESS
```

Comment:
should we keep the default value for `FROM_ADDRESS` here? and if required, user can override it.
For CCCP use case, we are going to have one unique address, if we keep the default value, we don't need to specify that every time we run the command.

Reply:
Specifying a default value could cause the new users to mistakenly use it without modifying values to suit their environment. That would cause mail notifications to not work. It might look as the service is not functioning as expected. Not specifying a default value would cause them to think about setting it up to suit their environment.

Command is going to quite surely end up in a script or ansible playbook. So I'm not worried about specifying it repeatedly. Plus, it needs to be specified only when spinning up seed-job and if we've done things correctly, there won't be a reason for us to re-create seed-job frequently.

Reply:
README can help clarify what needs to be set for parameters. Given the notifications are not working, openshift job error can clearly state that.

The service's primary user is CCCP, we should look for setting default values IMO.